### DISCUSSION TITLE
Do we have a golang version for this library?

### DISCUSSION THREAD

**Discussion by jiyanhbo** [aws/aws-advanced-jdbc-wrapper#460] [Q&A] [answered]
as title. Our system have both java & golang services accessing Postgres db. I am wondering if there is a golang library to handle the db failover. If not, is there any other golang library that can handle failover?

**Comment by crystall-bitquill** [comment, marked as answer]
Hi @jiyanhbo, thanks for your question. Unfortunately, there is no Golang version of this library available to handle failover and we are unaware of other libraries that support this functionality. We'll share this feedback with the team to help prioritize future development. However, have you evaluated the usage of [Amazon RDS Proxy](https://aws.amazon.com/rds/proxy/) to see if it can work with your scenario?
